tests/test_energy_overflows.py

```diff
@@ -272,13 +271,8 @@ def test_energy_overflows_with_summation_of_energies(precision):
     )
     nonbonded_gpu = nonbonded.to_gpu(precision)
 
-    # TBD: Remove the try/finally to clean up the GPU potential, else calling reset_device
-    # will fail
-    try:
-        # The reference should return a large energy value
-        assert nonbonded(x, nb_params, box) > 0.0
+    # The reference should return a large energy value
+    assert nonbonded(x, nb_params, box) > 0.0
```

Comment:
Can this assertion be strengthened? (comment says "large", but only tests "positive")

Reply:
Strengthened in https://github.com/proteneer/timemachine/pull/1090/commits/13988bcd6633fa0d7391a657052b3f1effc7e114